Jenkinsfile

```diff
@@ -782,7 +782,9 @@ pipeline {
                                          valgrind_stash: 'el8-gcc-nlt-memcheck'
                             recordIssues enabledForFailure: true,
                                          failOnError: false,
-                                         ignoreFailedBuilds: true,
+                                         // parameter below removed with the warnings-ng 11.2.2
+                                         // possibly by mistake.
```

Comment:
https://github.com/jenkinsci/warnings-ng-plugin/releases/tag/v11.0.0 states that the change in API was deliberate.

Reply:
Seems like we should the change be to just remove the parameter.

Reply:
Yes, I think that would be best in the short term.